### PR TITLE
Style cleanups

### PR DESCRIPTION
**Pull request by stsquad**
A couple of tweaks and clean ups for you.

### CODE REVIEW

**Comment by omar-polo** [comment]
I was about to open a very similar PR, fortunately I checked before posting :)

Just a thought: would you mind adding an autoload for `mu4e-dashboard-mode`, `mu4e-dashboard-activate` and `mu4e-dashboard-toggle`?  (to avoid creating another PR, since you're also adding a toggle function.)

**Comment by stsquad** [comment]
@omar-polo done

File: dashboard.org [hunk 1]
```diff
@@ -53,9 +53,9 @@ Call [[elisp:mu4e-dashboard-deactivate][mu4e-dashboard-deactivate]] before editi
 #+STARTUP: showall showstars indent
 
 #+NAME: query
-#+BEGIN_SRC shell :results list raw :var query="flag:unread count=5 
+#+BEGIN_SRC sh :results list raw :var query="flag:unread" count=5
```
Comment (rougier) [inline dashboard.org:56]: I thought we we supposed to use `shell` now instead of `sh`.

**Comment by rougier** [comment]
Many thanks !

File: dashboard.org (hunk 1 above)
Comment (stsquad) [reply dashboard.org:56]: "shell" is the user configured $SHELL whereas "sh" is /bin/sh which should always be a POSIX compliant shell. You can actually use any installed shell if it's setup in the org languages section.

Comment (rougier) [reply dashboard.org:56]: Ok. I think I got confused by `ob-sh` being renamed `ob-shell` and thus I assumed `shell` was the proper way.

**Pull request merged**